test/seqan3-test.cmake

```
@@ -91,6 +91,49 @@ list (APPEND SEQAN3_EXTERNAL_PROJECT_CMAKE_ARGS "-DCMAKE_INSTALL_PREFIX=${PROJEC
 # Commonly used macros for the different test modules in seqan3.
 # ----------------------------------------------------------------------------
 
+# e.g. target_source_file = "range/view/take.cpp"
+# component:
+#  * TARGET_NAME - the target name (e.g. "take")
+#  * TEST_NAME - the test name with target_path included (e.g. "range/view/take")
+#  * TARGET_PATH - the path to the target source (e.g. "range/view")
+macro (seqan3_test_component VAR _target_source_file _component_name_)
+    string (TOUPPER "${_component_name_}" _component_name)
```

```suggestion
    string (TOUPPER "${component_name_}" component_name)
```
too many underscores to keep track of :eyes: 

You did not add underscores for e.g. `target_name` in the other cmake code at all so maybe we stick to it?